packages/data/lib/src/remote/interceptors/tmdb_api_interceptor.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:data/src/key_store/store.dart';
+import 'package:data/src/request/tmdb_api/constants/query.dart';
+import 'package:dio/dio.dart';
+
+class TmdbApiInterceptor extends Interceptor {
+  final ApiKeyStore apiKeyStore;
+
+  TmdbApiInterceptor({required this.apiKeyStore});
```

Comment:
A constant constructor can't call a non-constant super constructor of 'Interceptor'.
I can implement interceptor class instead of extending, to make const constructor, but I will be forced to override additional 2 methods.